pkg/controller/cronjob/cronjob_controllerv2.go

```diff
@@ -517,16 +527,22 @@ func syncOne2(
 		t := nextScheduledTimeDuration(sched, now)
 		return t, nil
 	}
+	if isJobInActiveList(&batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%s-%d", cj.Name, getTimeHashInMinutes(scheduledTime)),
```

Comment:
Since you're doing that in two places it might be worth making this a method.

Reply:
done